src/test/java/Homework19.java

```diff
@@ -11,13 +12,11 @@ public void deletePlaylist() throws InterruptedException{
     login("jenyag5@gmail.com", "te$t$tudent");
     playlist();
     verifyPlaylistDeleted();
-    Thread.sleep(2000);
 }
   public void playlist() throws InterruptedException {
       WebElement songs = driver.findElement(By.cssSelector("#playlists > ul > li:nth-child(5) > a"));
        songs.click();
-       Thread.sleep(2000);
-       WebElement delete = driver.findElement(By.xpath("//*[@id=\"playlistWrapper\"]/header/div[3]/span/button"));
+       WebElement delete = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//*[@id=\"playlistWrapper\"]/header/div[3]/span/button")));
```

Comment:
**Hint**

How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)